py3-ply.yaml

```diff
@@ -14,15 +14,16 @@ vars:
 var-transforms:
   - from: ${{package.version}}
     match: _git(.*)
-    replace: ""
+    replace: ''
```

Comment:
Just remove this? I guess the script caught it?

Reply:
I dont' know why py-ruamel did that.

Reply:
it removed all the quoted string so I assume it just has a rule to unquote strings and then quote this as a char? make sense to me if a bit weird.